tests/yam/migration/setup_upgrade_env.pm

```diff
@@ -24,12 +24,17 @@ sub run {
     # Save the original value of the variables in order to restore it later if needed
     set_var('VERSION_ENV', get_var('VERSION'));
     set_var('SCC_ADDONS_ENV', get_var('SCC_ADDONS'));
+    set_var('AGAMA_ENV', get_var('AGAMA'));
```

Comment:
you need a loop here and check with if each one exist, otherwise you would set variable to 0 in maintenance unconditially.

Reply:
updated